src/Middleware/InitStateMiddleware.php

```diff
@@ -23,7 +23,6 @@ class InitStateMiddleware implements HTTPMiddleware
      */
     private static $admin_url_paths = [
         'dev/',
-        'graphql/',
```

Comment:
You need a hook in whatever consumes this config in order to prevent issues that would block people who _do_ use GraphQL with e.g. custom features in their CMS.

Reply:
It's private static config, people can update it in their project yml